.github/CODEOWNERS

```diff
@@ -18,14 +18,10 @@ CHANGELOG.md
 /crates/sui-graphql-rpc/ @oxade @amnn @wlmyng @stefan-mysten @emmazzz @suiwombat
 /crates/sui-graphql-e2e-tests/ @oxade @amnn @wlmyng @stefan-mysten @emmazzz @suiwombat
 
-
-# Notify Narwhal changes to interested folks.
-/narwhal/crypto/ @joyqvq @kchalkias
-/narwhal/executor/ @asonnino @akichidis @laura-makdah
-/narwhal/network/ @bmwill
-/narwhal/storage/ @akichidis
-
-/crates/sui-core/src/consensus_adapter.rs @laura-makdah
+# Consensus working group
+/consensus/ @akichidis @mwtian @arun-koshy
```

Comment:
Can you try @MystenLabs/consensus instead?

Reply:
would it be possible to migrate the consensus crates into /crates with the rest of the crates at some point?

Reply:
Probably not. There are still `external-crates/` and `sui-execution`.